.github/workflows/test.yml

```
@@ -0,0 +1,15 @@
+name: Test SDK
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)